src/__tests__/__snapshots__/theme.js.snap

```diff
@@ -15,6 +15,100 @@ Object {
     "48em",
     "64em",
   ],
+  "colorStyles": Object {
+    "blueOnLightBlue": Object {
+      "backgroundColor": "#cdf",
+      "color": "#007aff",
+    },
+    "blueOnWhite": Object {
+      "backgroundColor": "#fff",
+      "color": "#007aff",
+    },
+    "danger": Object {
```

Comment:
I don't see this danger and info in the actual source code. May need to regenerate snapshots